.github/CONTRIBUTING.md

```diff
@@ -42,21 +42,32 @@ Gradle uses pull requests for contributions. Fork [gradle/gradle](https://github
     git config user.name 'First Last'
     git config user.email user@example.com
 
+### IntelliJ
+
 You can generate the IntelliJ project by running
 
-    ./gradlew idea
+    gradlew idea
+
+### Eclipse
 
-You can generate the Eclipse projects by running
+You can generate the Eclipse project <ith the following steps
+  
+1. Install Eclipse 4.5 (Mars) at least
+2. Install the Groovy Eclipse plugin from http://dist.springsource.org/snapshot/GRECLIPSE/e4.5/
+3. Make sure you have a Java 8 compatible JDK configured in your workspace
+4. In `Window->Preferences->Groovy->Compiler`, check `Enable Script folder support` and add `**/*.gradle`
+5. Run `gradlew eclipse` from the root directory
```

Comment:
Please add `./`

Reply:
Adding './' is working for Linux but not for Windows. The command line for Windows should be either `.\gradlew`, either `gradlew`. Before the refactoring of @oehme , both syntax were displayed. So the change should be:
```
On Windows: gradlew
On Linux/OSX: ./gradlew
```
But this change should be applied everywhere... Do you want this?

Reply:
I think on Windows it'd be `gradlew.bat`. This is another case where we cannot guarantee a smooth contributor experience developing on Windows. 

I think we should state this and note that users should replace `./gradlew` with `gradlew.bat` for Windows.

Reply:
On Windows, the command executor aka 'cmd.exe' takes `gradlew` as well as `gradle.bat` and usually, Windows people do not add the suffix '.bat' to invoke a 'bat' command as below
```
C:\Users\fandre\Documents\git\gradle-poco>dir gradle*
 Le volume dans le lecteur C n’a pas de nom.
 Le numéro de série du volume est FCD5-3FB3

 Répertoire de C:\Users\fandre\Documents\git\gradle-poco

30/11/2017  13:43    <DIR>          gradle
14/10/2017  13:08            63 019 gradle.png
14/10/2017  13:08               126 gradle.properties
14/10/2017  13:09             5 326 gradlew
14/10/2017  13:22             2 290 gradlew.bat
01/12/2017  15:58    <DIR>          .gradle
               4 fichier(s)           70 761 octets
               2 Rép(s)  129 270 222 848 octets libres

C:\Users\fandre\Documents\git\gradle-poco>gradlew -version

------------------------------------------------------------
Gradle 4.4-rc-5
------------------------------------------------------------

Build time:   2017-11-30 15:21:37 UTC
Revision:     f4daa310b16df9abd4f37257bae8312b5293a251

Groovy:       2.4.12
Ant:          Apache Ant(TM) version 1.9.9 compiled on February 2 2017
JVM:          1.8.0_141 (Oracle Corporation 25.141-b15)
OS:           Windows 10 10.0 amd64

C:\Users\fandre\Documents\git\gradle-poco>gradlew.bat -version

------------------------------------------------------------
Gradle 4.4-rc-5
------------------------------------------------------------

Build time:   2017-11-30 15:21:37 UTC
Revision:     f4daa310b16df9abd4f37257bae8312b5293a251

Groovy:       2.4.12
Ant:          Apache Ant(TM) version 1.9.9 compiled on February 2 2017
JVM:          1.8.0_141 (Oracle Corporation 25.141-b15)
OS:           Windows 10 10.0 amd64

C:\Users\fandre\Documents\git\gradle-poco>.\gradlew.bat -version

------------------------------------------------------------
Gradle 4.4-rc-5
------------------------------------------------------------

Build time:   2017-11-30 15:21:37 UTC
Revision:     f4daa310b16df9abd4f37257bae8312b5293a251

Groovy:       2.4.12
Ant:          Apache Ant(TM) version 1.9.9 compiled on February 2 2017
JVM:          1.8.0_141 (Oracle Corporation 25.141-b15)
OS:           Windows 10 10.0 amd64

C:\Users\fandre\Documents\git\gradle-poco>./gradlew.bat -version
'.' n’est pas reconnu en tant que commande interne
ou externe, un programme exécutable ou un fichier de commandes.

C:\Users\fandre\Documents\git\gradle-poco>./gradlew -version
'.' n’est pas reconnu en tant que commande interne
ou externe, un programme exécutable ou un fichier de commandes.